crates/network/src/lib.rs

```diff
@@ -40,6 +40,13 @@ pub const PLAYER_INPUT_DATAGRAM_ID: u32 = 5;
 pub const WASM_DATAGRAM_ID: u32 = 6;
 
 components!("network", {
+    /// The absolute asset url, synced as a component between client and server for loading assets.
+    @[
+        Debuggable, Networked, Resource,
+        Name["Absolute Asset Url"],
+        Description["URL that is absolute, and can be used to load assets from the client or server."]
+    ]
+    abs_asset_url: String,
```

Comment:
Hm so the naming here feels weird; `AbsAssetUrl` is already a struct which is very different from this; the `AbsAssetUrl` struct is just a wrapper for a Url which point to an asset anywhere on the internet (and it can't be a relative path).

Something like `content_base_url` would make more sense to me.